src/coreclr/jit/hwintrinsicarm64.cpp

```diff
@@ -552,6 +552,11 @@ void HWIntrinsicInfo::lookupImmBounds(
                 immUpperBound = 7;
                 break;
 
+            case NI_Sve2_XorRotateRight:
+                immLowerBound = 1;
+                immUpperBound = genTypeSize(baseType) * BITS_PER_BYTE;
+                break;
```

Comment:
This is not needed. It will be handled by `else if (category == HW_Category_ShiftRightByImmediate)` logic above.